packages/commonwealth/server/routes/comments/getComments.ts

```diff
@@ -43,6 +44,10 @@ export const getComments = async (
     where['address_id'] = { [Op.in]: addressIds.map((p) => p.id) };
   }
 
+  if (thread_ids) {
```

Comment:
thanks @kurtisassad for the change
user can send multiple `thread_ids` too?
please share sample usage will it be something like 1,2,3.

Just wondering if we should add order by here, as we are returning data from multiple threads! Would be good for all scenarios!
```
models.Comment.findAndCountAll -> order by thread_id
```
Thanks!

Reply:
thanks sure, `create_time` sounds good.
is it desc(latest time first) or oldest time first(asc)
i believe `asc` will work good with pagination based on your comment above. Thanks!